shared/presentation/src/androidUnitTest/kotlin/network/bisq/mobile/presentation/offerbook/OfferbookPresenterFilterTest.kt

```
@@ -492,6 +492,129 @@ class OfferbookPresenterFilterTest {
+            every { offerUserProfileService.selectedUserProfile } returns MutableStateFlow(me)
+            coEvery { offerUserProfileService.isUserIgnored(any()) } returns false
+            coEvery { offerUserProfileService.getUserProfileIcon(any(), any()) } returns mockk(relaxed = true)
+            coEvery { offerUserProfileService.getUserProfileIcon(any()) } returns mockk(relaxed = true)
+
+            val marketPriceServiceFacade =
+                object : MarketPriceServiceFacade(mockk(relaxed = true)) {
+                    override fun findMarketPriceItem(marketVO: MarketVO) = null
+
+                    override fun findUSDMarketPriceItem() = null
+
+                    override fun refreshSelectedFormattedMarketPrice() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)